### DISCUSSION TITLE
Welcome to tuya-convert Discussions!

### DISCUSSION THREAD

**Discussion by kueblc** [ct-Open-Source/tuya-convert#852] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WulfgarCA** [comment]
Hi Colin,
realy like the tuya convert tool which helped me on my devices. Now I am seeing more and more updated firmwares where tuya connect fails, i usually just return those on Amazon.
I did get a shipment of 10 Smarlife bulbs from ali express now, and the first one failed during the process with a:
ID: 0142416f68626d64366147393149465231af28c06ddc538fb56a7ef15c679da0065f31f843807dee4c92f69d7d4a3cb611f6
PSK: fe44240cf91e6fa6c81358e5deefba17331843654977dfe912a0cd0aa833c7d7

I tried both the measter and the development branch.
I saw you were talking quite a bit about ID02 not sure if that matters here.

My question is, how can i best contribute. I was going to test out all 10 bulbs, but maybe one actually works, so i wanted to get prepared to capture as much information as possible for you guys.
Is it best to follow the procedure part of the collaboration document?

Thanks and cheers
Sascha

Edit: Forgot the Intro. Sascha 46 years old German living in San Jose, CA. It all started with christmas lights for my house and  I wanted to use Home Assitant and WLED.

Edit2: Those loghtbulbs are actually nit esp8622 based :(
